### ISSUE TITLE
When using gatsby's own gatsby plyugin google analytics GraphQL queries are affected

### ISSUE THREAD

**Issue by bhuwan1998** [konstantinmuenster/gatsby-starter-portfolio-minimal-theme#23]
I am trying to add google analytics tracking to the website but there's an issue whenever I include the plugin in gatsby config. 
```
module.exports = {
  plugins: [
    {
      resolve: "gatsby-theme-portfolio-minimal",
      options: {
        siteUrl: "https://www.bhuwanbansal.com", // Used for sitemap generation
        manifestSettings: {
          favicon: "./content/images/favicon.png", // Path is relative to the root
          siteName: " ", // Used in manifest.json
          shortName: " ", // Used in manifest.json
          startUrl: "/", // Used in manifest.json
          backgroundColor: "#FFFFFF", // Used in manifest.json
          themeColor: "#000000", // Used in manifest.json
          display: "minimal-ui", // Used in manifest.json
        },
        contentDirectory: "./content",
        blogSettings: {
          path: "/blog", // Defines the slug for the blog listing page
          usePathPrefixForArticles: true, // Default true (i.e. path will be /blog/first-article)
        },
      },
      // resolve: `gatsby-plugin-google-analytics`,
      // options: {
      //   // The property ID; the tracking code won't be generated without it
      //   trackingId: "tracking-id", // GA
      //   head: true,
      // },
    },
    "gatsby-plugin-image",
    "gatsby-plugin-sharp",
    "gatsby-transformer-sharp", // Needed for dynamic images
  ],
};

```

<img width="1046" alt="image" src="https://github.com/konstantinmuenster/gatsby-starter-portfolio-minimal-theme/assets/24411210/920abf64-3a5a-4aa9-8fdc-9e6af7be3fe2">

I am confused why this is happening and need some guidance understanding how to fix this. Any help with this will be appreciated.

**Comment by konstantinruge** [comment]
Hey @bhuwan1998! The theme itself includes a Google Analytics integration. So you don't need to add the plugin manually yourself.

Have you tried if setting the Google Analytics options for the theme (as in the [docs](https://github.com/konstantinmuenster/gatsby-theme-portfolio-minimal/tree/main/gatsby-theme-portfolio-minimal#configuring-gatsby-configjs)) works?

**Comment by bhuwan1998** [comment]
I think I may have removed the included integration early on. I will look at the files and update here soon.

**Comment by bhuwan1998** [comment]
I have re-implemented the in-built google analytics within the theme but I am unsure of how to actually implement it even though there is only one argument to fill in which is the measurement ID for the website. I have filled it in. However, there is no traffic shown or no connection with google analytics at the moment. In the screenshot the measurementID/trackingID is not mentioned for obvious reasons.

Any guidance would be appreciated. Let me know if there is any additional information required. 

<img width="282" alt="Google Analytics SC" src="https://github.com/konstantinmuenster/gatsby-starter-portfolio-minimal-theme/assets/24411210/ac13acf6-753a-46dc-a4c2-71850f8226bb">

**Comment by konstantinruge** [comment]
Thanks for the update. I might need to look into it then. It doesn't work on the deployed site either, right? (since you only specified `production` as the target environment)

**Comment by bhuwan1998** [comment]
I initially had both development and production. But since my website is deployed I thought I can just use production. The website is deployed on netlify at this point. You can the view the website here - [website link](https://www.bhuwanbansal.com)

**Comment by konstantinruge** [comment]
Alright, I will look into it 👍  The website looks great, btw 😁

**Comment by bhuwan1998** [comment]
So at the moment in the in-built one analytics is not working and when I use the gatsby plugin the graphql queries fail on testing development.

**Comment by bhuwan1998** [comment]
> Alright, I will look into it 👍 The website looks great, btw 😁

Thanks. All thanks to your theme

**Comment by konstantinruge** [comment]
@bhuwan1998 I released a new version (4.6.3) - this should hopefully fix the issue 🤞
